src/main/java/uk/gov/laa/ccms/caab/controller/requests/ProviderRequestsController.java

```diff
@@ -404,7 +404,7 @@ protected String providerRequestsDetails(
     populateProviderRequestDetailsLookupDropdowns(model, dynamicForm);
 
     providerRequestDetailsForm.setClaimUploadEnabled(
-        dynamicForm.getIsClaimUploadEnabled());
+        Boolean.TRUE.equals(dynamicForm.getIsClaimUploadEnabled()));
```

Comment:
Is this for null handling?

Reply:
Yeah, due to not having everything running locally this was returning null and causing a `NullPointerException` so I made it null safe.